chapters/syntax.tex

```diff
@@ -27,21 +27,23 @@ \section{Lexical conventions}\doublelabel{lexical-conventions}
 DIGIT = "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
 UNSIGNED-INTEGER = DIGIT { DIGIT }
 UNSIGNED-NUMBER = UNSIGNED-INTEGER [ "." [ UNSIGNED-INTEGER ] ]
-        [ ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER ]
+        [ ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER ] | 
+         "."  UNSIGNED-INTEGER [ ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER ]
```

Comment:
This doesn't look like the grammar change we polled on.  Looks like this accepts strange things like `0.13.0` or even `0..0`, while still accepting `13` (the original issue).

Reply:
I don't see how `0.13.0` or `0..0` would be allowed by it, as `|` has the lowest priority.

However, there were two grammar changes polled for, and it seems I only included the 2nd one (allowing .13), but not the first one (forbidding 13 while allowing 13.).

I have now included the first one as well.

Reply:
Hans, perhaps move the `|` to the beginning of the next line, where it is less likely to be overlooked.

However, I don't think this corresponds with the decisions made in #2122 yet, since it still accepts `13`, and it seems we were going to call this `REAL-UNSIGNED-NUMBER`.